css/elementor-icons.css

```diff
@@ -1875,7 +1875,7 @@
   content: "\e9ca";
 }
 
-.eicon-ehp-zigzage:before {
+.eicon-ehp-zigzag:before {
```

Comment:
@nuritsha Can you please check if this needs to be updated in `css/elementor-icons.min.css` as well? (run grunt)